go/vt/servenv/servenv_test.go

```diff
@@ -65,9 +65,7 @@ func TestFireOnCloseHooksTimeout(t *testing.T) {
 		time.Sleep(1 * time.Second)
 	})
 
-	// we deliberatly test the flag to make sure it's not accidently set to a
-	// high value.
-	if finished, want := fireOnCloseHooks(onCloseTimeout), false; finished != want {
+	if finished, want := fireOnCloseHooks(1*time.Nanosecond), false; finished != want {
```

Comment:
1ns is here to check backward compact. given the default value is change now ?

Reply:
I made this change, because otherwise the test would need to run for more than 10 seconds. The intent of the test is to verify that `fireOnCloseHooks` respects the timeout provided to it and that it returns after that much time. If we give the default here, we would need a sleep for like 11 seconds or something in the Onclose function and that would just slow down the test.